src/binary/ibc-setup/commands/balances.ts

```diff
@@ -76,9 +77,9 @@ export async function run(options: Options) {
     .join(os.EOL);
 
   if (!balances) {
-    console.log('No funds found for default denomination on any chain.');
+    logger.info('No funds found for default denomination on any chain.');
```

Comment:
I would actually use consol.log on these pieces. The stuff that is supposed to go to stdout for interactive commands.  All the details of running should go to logger